apps/web/app/api/outlook/linking/callback/route.ts

```diff
@@ -116,15 +115,15 @@ export const GET = withError(async (request: NextRequest) => {
 
     if (!existingAccount) {
       logger.warn(
-        `Merge Failed: Microsoft account ${providerEmail} (${providerAccountId}) not found in the system. Cannot merge.`,
+        `Merge Failed: Microsoft account ${providerEmail} not found in the system. Cannot merge.`,
```

Comment:
Best way to do this is:

```ts
warn(
`Merge Failed: Microsoft account not found in the system. Cannot merge.`,
{ email: providerEmail })
```

It makes searching logs much easier as it's properly structured and you have consistent strings.

Reply:
this comes up 5 times in this file